optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/DefaultSolverTest.java

```diff
@@ -105,9 +107,10 @@ public void solveMetrics() {
         assertThat(solution).isNotNull();
         assertThat(solution.getScore().isSolutionInitialized()).isTrue();
 
-        assertThat(meterRegistry.getMeasurement("optaplanner.solver.solve-length", "DURATION")).isEqualTo("0.0");
-        assertThat(meterRegistry.getMeasurement("optaplanner.solver.solve-length", "ACTIVE_TASKS")).isEqualTo("0.0");
-        assertThat(meterRegistry.getMeasurement("optaplanner.solver.errors", "COUNT")).isEqualTo("0.0");
+        assertThat(meterRegistry.getMeasurement("optaplanner.solver.solve-length", "DURATION").intValueExact()).isEqualTo(0);
+        assertThat(meterRegistry.getMeasurement("optaplanner.solver.solve-length", "ACTIVE_TASKS").intValueExact())
+                .isEqualTo(0);
+        assertThat(meterRegistry.getMeasurement("optaplanner.solver.errors", "COUNT").intValueExact()).isEqualTo(0);
```

Comment:
AssertJ has `isZero()` - you should be able to avoid calling `BigDecimal.intValueExact()`.